app/Platform.php

```diff
@@ -24,6 +24,16 @@ public static function normalizePath(string $path): string
         return $path;
     }
 
+    public static function getGlobalComposerHome(): string
+    {
+        return rtrim(shell_exec('composer -n global config home'), "\n");
```

Comment:
```suggestion
        return rtrim(shell_exec('composer -n global config home --quiet'), "\n");
```

Reply:
I can't see a difference in output when using the `quiet` option. Does it actually even do anything?

Reply:
Yeah it does actually, on my case it's showing some deprecation warnings, so with `--quiet` it's just showing the wanted result.

Reply:
oh ok, cool. I'll add that in